src/components/inputs/checkbox-input/checkbox.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
Why is the module called `checkbox.js` and not `checkbox-input.js`?

Reply:
🤣 because the other file is called 'checkbox-input.js'

Reply:
Ah right. Thanks